Server/JokesOnYou.Web.Api/Data/DataSeeding.cs

```diff
@@ -15,18 +15,20 @@ public static void Initialize(DataContext context)
         {
             context.Database.EnsureCreated(); //Make sure db actually exists
 
-            // Look for any students.
+            // Look for any users in db
             if (context.Users.Any())
             {
+                //If there are students then leave method
```

Comment:
you can remove this comment or if you want to keep it change students to users

Reply:
idk why but i keep writing students instead of users, xd